src/puppetlabs/puppetdb/command.clj

```diff
@@ -615,7 +624,11 @@
              :response-chan response-chan
              :response-mult response-mult
              :response-chan-for-pub response-chan-for-pub
-             :response-pub (async/pub response-chan-for-pub :id))))
+             :response-pub (async/pub response-chan-for-pub :id)
+             ;; This coordination is needed until we no longer
+             ;; support jdk < 10.
+             ;; https://bugs.openjdk.java.net/browse/JDK-8176254
+             :activities (atom {:executing-delayed 0}))))
```

Comment:
maybe give the two activities atoms names that are specific to what they're keeping track of